tools/performance/PowerShellPerf.wprp

```diff
@@ -0,0 +1,148 @@
+<?xml version='1.0' encoding='utf-8' standalone='yes'?>
+<WindowsPerformanceRecorder
+    Author="PowerShell Performance"
+    Comments="PowerShell Performance"
+    Company="Microsoft Corporation"
+    Copyright="Microsoft Corporation"
+    Team="PowerShell Performance"
+    Version="1.0"
+    >
+  <Profiles>
+    <SystemCollector Id="SystemCollector" Name="NT Kernel Logger">
```

Comment:
Despite what the outdated MSDN documentation says, the Name attribute has been optional since Windows 8.1, and should be considered deprecated.

There's probably no value targeting the WPR that ships with the Windows 8 ADK as opposed to the Windows 8.1 ADK. Newer versions of the ADK can be installed on older OS releases.

Reply:
Thanks, I appreciate the review. I agree, folks profiling PowerShell probably aren't running Win8 so I'll just remove the Name attribute.